libs/http.liq

```diff
@@ -132,7 +132,9 @@ def harbor.http.static.base(serve,~content_type,~basepath,~port,~headers,directo
   end
 
   def register(method)
-    serve(port=port,method=method,"^#{basepath}",handler(method=method))
+    # TODO: restore me...
+    # serve(port=port,method=method,"^#{basepath}",handler(method=method))
```

Comment:
This seems unrelated?

Reply:
It was related: the change induced a subtle typing problem related to the fact that we have to correctly handle variance now that we have non-ground types. This is fixed anyway.